### PR TITLE
Translate 2022 PWA chapter for Simplified Chinese

### PR DESCRIPTION
**Pull request by ibelem**
Part of #1067

* Fix HTTP 404 for "why it isn’t trivial to find out actual usage of service worker" link in https://almanac.httparchive.org/en/2022/pwa#workbox-usage
* Translate 2022 PWA chapter for Simplified Chinese 

@tunetheweb 

PTAL.

### CODE REVIEW

**Comment by ibelem** [comment]
@rviscomi Could you please help to enable the CI again then I can check if the linting issues were fixed? Many thanks! 😊

File: src/content/zh-CN/2022/pwa.md [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+
+## 总结
+
+2022 年对于 PWA 来说是辉煌的一年。允许将可安装 Web 应用与桌面平台集成的功能不断增加，推动了该技术被业内知名厂商采用。在过去的一年里，协议处理（protocol handlers）、窗口控件覆盖（window controls overlay）、在系统登录时运行等高级功能已将 PWA 定位为应用开发的关键技术。虽然令人鼓舞，但这并不代表整个 Web 平台。与 2021 年的数据相比，Service Worker 的使用百分比下降到一半左右，但使用 PWA 技术构建的大型应用正在兴起。
+
+Manifest 文件继续处于健康状态，桌面端的比例比去年略有增加，达到了 95%。这些文件的正确性非常好，但它们的完整性仍有诸多不足。目前，只有大约 0.8% 的网站符合可安装的条件。许多高级功能，如 `shortcuts`  和 `share_target` 开始受到关注，出现在大约 5% 的 PWA 中。`protocol_handlers` 和窗口控件覆盖等功能太新，暂时无法对数据产生影响。今年还提供了许多 Fugu API 的初始快照。
+
+可以理解，通知疲劳仍然是一个因素，但用户也会请求并欣赏合法的通知用例。浏览器厂商正在尝试侵入性较低的权限请求，而 Web 推送通知的优势在于提供跨平台的一致体验，从而为用户提供他们所请求的与设备无关的轻推。
+
+我们希望这些信息能为您的 PWA 之旅提供一些启示，并帮助开发人员了解在 API 适配方面的技术趋势。
```
Comment (tunetheweb) [inline src/content/zh-CN/2022/pwa.md:524]:
```suggestion
我们希望这些信息能为您的 PWA 之旅提供一些启示，并帮助开发人员了解在 API 适配方面的技术趋势。

```

File: src/content/zh-CN/2022/pwa.md [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+featured_stat_label_2: 桌面端 PWA 使用 Service Worker 的 install 事件
+featured_stat_3: 60%
+featured_stat_label_3: 以某种方式使用 Workbox 的 PWA 应用
+---
+
+## 简介
+
+在渐进式 Web 应用 PWA 的早期，有两个关键功能吸纳了现代 Web 应用的能力，离线支持和添加到主屏幕的快捷方式。
+
+这两个概念在安装 PWA 后启用，该过程通常从点击浏览器地址栏中出现的安装图标开始。该图标会提示用户安装该网站。Samsung Internet 和 Mozilla Firefox 等移动浏览器最早明确支持此行为，被熟知为<a hreflang="zh-CN" href="https://developer.mozilla.org/zh-CN/docs/Web/Progressive_web_apps/Add_to_home_screen">`添加到主屏幕 (A2HS)`</a>。
```
Comment (tunetheweb) [inline src/content/zh-CN/2022/pwa.md:25]: For MDN we will automatically not the language based on the URLs to no need to convert to HTML links. Also I don't think this should be in code format?

```suggestion
这两个概念在安装 PWA 后启用，该过程通常从点击浏览器地址栏中出现的安装图标开始。该图标会提示用户安装该网站。Samsung Internet 和 Mozilla Firefox 等移动浏览器最早明确支持此行为，被熟知为[添加到主屏幕 (A2HS)](https://developer.mozilla.org/zh-CN/docs/Web/Progressive_web_apps/Add_to_home_screen)。
```

File: src/content/zh-CN/2022/pwa.md [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+
+五年前，这还是一个激进的想法。一个网站能够直接从主屏启动，并与设备中其他安装的应用一起列出。这是在缩小 Web 应用的功能与特定于操作系统体验之间差距的方面取得进展的开始。
+
+Web 应用的添加到主屏幕的场景已经演变为，可以完全在移动和桌面环境中安装，并深度集成到了操作系统中。在过去的 12 个月里，浏览器在确保 PWA 与桌面平台紧密集成方面取得了重要进展，今年年鉴中的许多新增内容都反映了这些变化。这是 2022 年 PWA 的最新状态。
+
+注：作为 Web 技术的集合，PWA 并不与 Web 平台的其余部分隔绝。前面的章节专门介绍了[能力](./capabilities)，今年我们也研究了在 PWA 中使用这些新的高级功能。
+
+## Service Workers
+
+<a hreflang="zh-CN" href="https://developer.mozilla.org/zh-CN/docs/Web/API/Service_Worker_API">Service Workers</a> 是 PWA 的核心技术之一，使离线应用、获取推送通知以及后台处理成为可能。Service Workers 是我们从应用中获得大多数高级体验的基石。Service Workers 还被用于定义数据更新和支持即将推出的新功能，例如 <a hreflang="en" href="https://github.com/aarongustafson/pwa-widgets#rich-widgets">基于 PWA 技术的 widgets</a>。
```
Comment (tunetheweb) [inline src/content/zh-CN/2022/pwa.md:35]:
```suggestion
[Service Workers](https://developer.mozilla.org/zh-CN/docs/Web/API/Service_Worker_API) 是 PWA 的核心技术之一，使离线应用、获取推送通知以及后台处理成为可能。Service Workers 是我们从应用中获得大多数高级体验的基石。Service Workers 还被用于定义数据更新和支持即将推出的新功能，例如 <a hreflang="en" href="https://github.com/aarongustafson/pwa-widgets#rich-widgets">基于 PWA 技术的 widgets</a>。
```

File: src/content/zh-CN/2022/pwa.md [hunk 4]
```diff
@@ -0,0 +1,524 @@
...
+
+在 Service Worker 功能支持方面，虽然各主要浏览器之间并不一致，但 Webkit 添加对<a hreflang="en" href="https://caniuse.com/push-api">推送通知</a> 的支持是一个巨大的里程碑。今年早些时候，Apple 宣布<a hreflang="en" href="https://webkit.org/blog/12945/meet-web-push/">对其桌面平台进行了更改</a>，以支持 <a hreflang="zh-CN" href="https://developer.mozilla.org/zh-CN/docs/Web/API/Push_API">Push API</a> 、<a hreflang="zh-CN" href="https://developer.mozilla.org/zh-CN/docs/Web/API/Notifications_API">Notifications API</a> 以及 <a hreflang="zh-CN" href="https://developer.mozilla.org/zh-CN/docs/Web/API/Service_Worker_API">Service Workers</a> 将会启用 Web Push。Apple 还宣布该功能将于 2023 年进入他们的移动平台。
```
Comment (tunetheweb) [inline src/content/zh-CN/2022/pwa.md:37]:
```suggestion
在 Service Worker 功能支持方面，虽然各主要浏览器之间并不一致，但 Webkit 添加对<a hreflang="en" href="https://caniuse.com/push-api">推送通知</a> 的支持是一个巨大的里程碑。今年早些时候，Apple 宣布<a hreflang="en" href="https://webkit.org/blog/12945/meet-web-push/">对其桌面平台进行了更改</a>，以支持 [Push API](https://developer.mozilla.org/zh-CN/docs/Web/API/Push_API) 、[Notifications API](https://developer.mozilla.org/zh-CN/docs/Web/API/Notifications_API) 以及 [Service Workers](https://developer.mozilla.org/zh-CN/docs/Web/API/Service_Worker_API) 将会启用 Web Push。Apple 还宣布该功能将于 2023 年进入他们的移动平台。
```

File: src/content/zh-CN/2022/pwa.md [hunk 5]
```diff
@@ -0,0 +1,524 @@
...
+  sheets_gid="2067971287",
+  sql_file="sw_adoption_over_time_ranking.sql"
+  )
+}}
+
+在前 1000 个网站中，支持 Service Worker 的页面数量没有明显的变化，桌面端略有下降，而移动端的增长则更小，但所有其他类别都有所增加。如果我们遵循[去年的推理](../2021/pwa#fig-2)，即假设大型网站更快地采用先进技术，那么看到其他类别的更多增长是合理的。似乎较小的公司和开发人员已经学习并采用了从流量更多的公司案例研究和示例中分享的技术。
+
+### Service Worker 事件
+
+Service Worker 在 Web 应用、浏览器和网络之间充当代理服务器的角色。要安装 Service Worker，必须获取并注册它。注册成功后，Service Worker 将在一个<a hreflang="zh-CN" href= "https://developer.mozilla.org/zh-CN/docs/Web/API/ServiceWorkerGlobalScope">特殊 Woker 容器</a>中执行，该容器在主线程之外运行并且没有 DOM 访问权。这是可以处理事件的时机。
```
Comment (tunetheweb) [inline src/content/zh-CN/2022/pwa.md:62]:
```suggestion
Service Worker 在 Web 应用、浏览器和网络之间充当代理服务器的角色。要安装 Service Worker，必须获取并注册它。注册成功后，Service Worker 将在一个[特殊 Woker 容器](https://developer.mozilla.org/zh-CN/docs/Web/API/ServiceWorkerGlobalScope)中执行，该容器在主线程之外运行并且没有 DOM 访问权。这是可以处理事件的时机。
```

**Review by tunetheweb — CHANGES_REQUESTED** [review]
LGTM apart from a note about MDN links.

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**